src/bitcoind.cpp

```diff
@@ -254,10 +253,18 @@ int main(int argc, char* argv[])
     util::WinCmdLineArgs winArgs;
     std::tie(argc, argv) = winArgs.get();
 #endif
+
+    NodeContext node;
+    int exit_status;
+    std::unique_ptr<interfaces::Init> init = interfaces::MakeNodeInit(node, argc, argv, exit_status);
+    if (!init) {
+        return exit_status;
+    }
```

Comment:
I apologize for revisiting this old PR, but I couldn't resist asking to clarify things for me.

As far as I understand, `interfaces::MakeNodeInit` returns a `std::unique_ptr` that owns an instance of `init::BitcoindInit` or throws an exception. Given this understanding, it seems unclear how the `return exit_status;` statement could ever execute.